formats/finder/publisher/details.json

```diff
@@ -12,15 +12,30 @@
       "type": "string",
       "additionalProperties": false
     },
+    "email_signup_enabled": {
+      "type": "boolean"
+    },
     "filter": {
+      "description": "This is the fixed filter that scopes the finder",
       "type": "object",
-      "additionalProperties": true
+      "additionalProperties": false,
+      "properties": {
```

Comment:
This should also include a "format" property (the [policies finder](https://github.com/alphagov/policy-publisher/blob/master/lib/policies_finder_publisher.rb#L62-L64) filters on `"format": "policy"`) otherwise the [spec for the finder](https://github.com/alphagov/policy-publisher/blob/master/spec/lib/policies_finder_publisher_spec.rb) will fail.